src/MainWindow.xaml.cs

```diff
@@ -34,6 +34,12 @@ public sealed partial class MainWindow : Window, INotifyPropertyChanged
     // Goes true on DispatcherQueue.ShutdownStarting
     static internal bool IsShuttingDown = false;
 
+    // MCP Server instance for serving package data via Model Context Protocol
+    private McpServerService _mcpServer;
+
+    // Flag to indicate if running in MCP server mode (headless)
+    private bool _isMcpServerMode = false;
```

Comment:
Rather than having this bool field, have a bool property that checks if _mcpServer is null or not